f/mapClickTeleport/fn_mapClickTeleportBriefing.sqf

```diff
@@ -0,0 +1,43 @@
+// F3 - Mission Maker Teleport
+// Credits: Please see the F3 online manual (http://www.ferstaberinde.com/f3/en/)
+// ====================================================================================
+
+params [["_unit", objNull, [objNull]]];
+
+if (!hasInterface) exitWith {};
+
+if (!isDedicated && (isNull player)) then
+{
+	waitUntil {sleep 0.1; !isNull player};
+};
+
+if (_unit != player) exitWith {};
+
+
+player createDiaryRecord ["diary", ["HALO Jumping 101",
```

Comment:
I'd prefer is this was actually under the briefing component but that's a future refactor.

Reply:
In my opinion, the briefing component shouldn't have to know (details about) any other component. It's not the briefing component's job to describe the details of how other components work?

Reply:
This will probably disappear when I get around to that briefing stringtable refactor but, while I agree, I think the briefing component should be the **only** component interacting with the diary.

Also, I want to test removing the async component init sometime because I think it actually causes more trouble than it's worth.

Anyway, this is future:tm: